test/typedarray/dataview.js

```diff
@@ -104,7 +103,7 @@ function testOneOffset(dataView, offSet, value)
                 print(getFuncs[j] + " = " +  result);
                 }
 
-            print("set little endian value offset " + offSet + " value " + value + " method " + setFuncs[i]);
+            print("set big endian value offset " + offSet + " value " + value + " method " + setFuncs[i]);

```

Comment:
You can ignore the changes to dataview.js, dataview.baseline and samethread.baseline. I fixed this typo in the comments of this test and that caused a lot of changes to the baseline. No other changes to these files.